Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/configuration/InternalExchangeManagementPortDescriptor.java

```diff
@@ -0,0 +1,26 @@
+package gov.hhs.fha.nhinc.configuration;
+
+import gov.hhs.fha.nhinc.internalexchangemanagement.EntityInternalExchangeManagementPortType;
+import gov.hhs.fha.nhinc.messaging.service.port.SOAP12ServicePortDescriptor;
+
+/**
+ *
+ * @author ptambellini
+ */
+public class InternalExchangeManagementPortDescriptor
+extends SOAP12ServicePortDescriptor<EntityInternalExchangeManagementPortType> {
+
+    private static final String NAMESPACE_URI = "urn:gov:hhs:fha:nhinc:internalexchangemangement";
+    private static final String WS_ADDRESSING_ACTION = NAMESPACE_URI + ":EntityInternalExchangeManagementRequest";
```

Comment:
Optional: Remove NAMESPACE_URI. I don't see any other user for NAMESPACE_URI apart from concatenating it with EntityInternalExchangeManagementRequest to get addressing action.